orchestrator/graphql/__init__.py

```diff
@@ -66,6 +68,9 @@ class Query:
         resolver=resolve_settings,
         description="Returns information about cache, workers, and global engine settings",
     )
+    customers: DefaultCustomerType = authenticated_field(
```

Comment:
I think we do want the customers as a `Connection` like the others, even with just 1 default customer

Reply:
Made some changes regarding this comment—let me know if what I did was in line with what you meant